scc/blue-green-with-packageinstall.hbs.md

```diff
@@ -191,18 +149,48 @@ After a new version of the package is added to the GitOps repository, create a n
     values:
     - secretRef:
         name: green-dev-values
-  ```
+```
+
+## Practicing Blue-Green Deployment
```

Comment:
Instead of Practicing, maybe we can use "Execute"? The word practice infers this is a safe operation (as in no consequences if you mess it up).

Or maybe we can use

> Divide traffic between the Blue and Green Deployments

Apologies for not having this suggestion ready in my initial feedback on the jira issue